### ISSUE TITLE
Squid3 v1.1.8: Incomplete amendment of Th (ppm) expression in Th-centric Tasks

### ISSUE THREAD

**Issue by sbodorkos** [CIRDLES/Squid#315]
Still a slight glitch in the evaluation of Th (ppm) in Th-centric Tasks, i.e. relating to the utilisation of "four horsemen" EqNum -4 when the primary concentration element is Th.

For **Unknowns**, the relevant expression "Th_Concen" is **correctly** specified as shown below:

![image](https://user-images.githubusercontent.com/17473203/56623918-a9f51d00-6679-11e9-9041-240cb568985b.png)

For **Reference Materials**, the expression **should** be completely identical, exploiting the fact that expression "ParentElement_ConcenConst" is RU-flagged. Unfortunately, inspection of the relevant expression "Th_Concen_RM" shows that the circular logic you highlighted recently (in a recently-closed issue: https://github.com/CIRDLES/Squid/issues/311#issuecomment-483636535) is still in place for Reference Material analyses:

![image](https://user-images.githubusercontent.com/17473203/56631416-8ee5d580-6698-11e9-91bc-4f811de97586.png)

So basically, the relevant code for Unknowns has been correctly remediated, but the analogous code for Reference Materials has not.

**Comment by bowring** [comment]
It appears you have not rebuilt the project by re-importing the task?  If you still get this result, then please send data and task file for use by me.  Cheers.
